shared/settings/landing/index.desktop.js

```diff
@@ -0,0 +1,264 @@
+    case 1:
+    default:
+      return {
+        type: 'upgrade',
+        onUpgrade: () => onUpgrade(otherLevel),
+      }
+  }
+}
+
+const Divider = () => <Box style={{height: 1, backgroundColor: globalColors.black_40, flex: 1}} />
+
+function Stars ({level}: {level: PlanLevel}) {
+  return null
+}
+
+function SpaceInfo ({freeSpace, freeSpacePercentage}: {freeSpace: string, freeSpacePercentage: number}) {
+  return (
+    <Text type={'BodySmall'}>
+      space info
+    </Text>
+  )
+}
+
+function UpgradeButton ({onUpgrade}: {onUpgrade: () => void}) {
+  return (
+    <Text type={'BodySmall'}>
+      upgrade
+    </Text>
+  )
+}
+
+function DowngradeLink ({onDowngrade}: {onDowngrade: () => void}) {
+  return (
+    <Text type={'BodySmall'}>
+      downgrade
+    </Text>
+  )
+}
+
+function PlanActionVariants ({variants}: {variants: PlanActionVariantsProps}) {
+  switch (variants.type) {
+    case 'downgrade':
+      return <DowngradeLink onDowngrade={variants.onDowngrade} />
+    case 'upgrade':
+      return <UpgradeButton onUpgrade={variants.onUpgrade} />
+    case 'spaceInfo':
+      return <SpaceInfo {...variants.spaceInfo} />
+  }
+}
+
+function PlanLevelRow ({level, onInfo, variants, style}: PlanLevelProps) {
+  const selected = variants.type === 'spaceInfo'
+
+  return (
+    <Box style={{...globalStyles.flexBoxRow, ...planLevelRowStyle, backgroundColor: selected ? globalColors.blue4 : globalColors.white, ...style}}>
+      <Box style={{...globalStyles.flexBoxRow}}>
+        <Text onClick={() => onInfo()} type={'BodyPrimaryLink'}>
+          {level}
+        </Text>
+        <Text style={{...globalStyles.flex}} type={'BodySmall'}>
+          ({levelToPrice[level]})
+        </Text>
+      </Box>
+      <Box style={{...globalStyles.flexBoxRow}}>
+        <Text style={{...globalStyles.flex}} type={'BodySmall'}>
+          ({levelToSpace[level]})
+        </Text>
+        <Stars level={level} />
+      </Box>
+      <Box style={{...globalStyles.flexBoxRow}}>
+        <PlanActionVariants variants={variants} />
+      </Box>
+    </Box>
+  )
+}
+
+function PaymentInfo ({name, last4Digits, isBroken, onChangePaymentInfo}: PaymentInfoType & {onChangePaymentInfo: () => void}) {
+  return (
+    <Box style={globalStyles.flexBoxColumn}>
+      <Text
+        style={{color: globalColors.black_40}}
+        type='BodySmall'>
+        Your payment method
+      </Text>
+      <Divider />
+      <Box style={{...globalStyles.flexBoxRow, justifyContent: 'space-between', alignItems: 'center'}}>
+        <Box style={globalStyles.flexBoxColumn}>
+          <Text
+            type='Body'>
+            {name}
+          </Text>
+          <Text
+            style={{color: isBroken ? globalColors.red : globalColors.black_40}}
+            type='BodySmall'>
+            **** {last4Digits} {isBroken ? ' (broken)' : ''}
+          </Text>
+        </Box>
+        <Button type='Primary' label='Update' onClick={onChangePaymentInfo} />
+      </Box>
+    </Box>
+
```

Comment:
I disabled eslint for that commit since it had a couple other errors that were going to get fixed in the next commit